### PR TITLE
add linux sample

### PR DESCRIPTION
**Pull request by danewalton**

### CODE REVIEW

**Comment by danewalton** [comment]
Holding until after preview release

File: demos/projects/PC/linux/ADU.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+If using WSL to build this project, it is **HIGHLY** recommended to clone the project to the file system inside the WSL instance. Do not clone the repo to your Windows file system and open the WSL working directory to a Windows formatted directory. Clone times, build times, directory max lengths, among other problems may arise.
+
+## Get the middleware
+
+Clone the following repo to download all sample device code, setup scripts, and offline versions of the documentation.
+
+**If you previously cloned this repo in another sample, you don't need to do it again.**
+
+```bash
+git clone https://github.com/Azure-Samples/iot-middleware-freertos-samples.git
```
Comment (ewertons) [inline demos/projects/PC/linux/ADU.md:30]: Is it missing the feature branch for now? (-b feature/iot-adu )

**Review by ewertons — APPROVED** [review]

File: demos/projects/PC/linux/ADU.md (hunk 1 above)
Comment (danewalton) [reply demos/projects/PC/linux/ADU.md:30]: Yes good call. Will add

**Review by vaavva — APPROVED** [review]

File: demos/projects/PC/linux/ADU.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+To deploy the update to your linux machine, follow the link below:
+
+- [Deploy Update](https://docs.microsoft.com/azure/iot-hub-device-update/deploy-update)
+
+After you deploy the update, the device should receive the new writable properties payload (i.e., the ADU service “request”) and start processing it.
+
+Once the device reboots, you should see on the console, output that looks like the following:
+
+![img](../../../../docs/resources/new-version-device-output.png)
```
[the remark this answers is not in the document]
Comment (danewalton) [reply demos/projects/PC/linux/ADU.md:209]: Good point this also doesn't reboot. Let me change this section.

File: demos/projects/PC/linux/ADU.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+```bash
+git clone https://github.com/Azure-Samples/iot-middleware-freertos-samples.git
+```
+
+To initialize the repo, run the following command:
+
+```bash
+cd iot-middleware-freertos-samples
+git checkout feature/iot-adu
```
[the remark this answers is not in the document]
Comment (danewalton) [reply demos/projects/PC/linux/ADU.md:37]: Yes since they could be viewing this on Github :/

**Pull request merged**